### PR TITLE
Add configuration files for Read the Docs (#infra)

### PR DESCRIPTION
**Pull request by poncovka**
Provide new configuration files for Read the Docs and its requirements.

(cherry-picked from a commit c9f0989)

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Thanks for porting the fix!

**Review by M4rtinK — APPROVED** [review]
Looks good to me as well, thanks! :)

**Review by jkonecny12 — APPROVED** [review]
Looks great to me. Thanks!

**Comment by jkonecny12** [comment]
@jstodola could you please ACK this? Thanks!

File: .structure-config [hunk 1]
```diff
@@ -1 +1 @@
-INFRASTRUCTURE_FILES='.structure-config .packit.yml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
+INFRASTRUCTURE_FILES='.structure-config .packit.yml .readthedocs.yaml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
```
Comment (jkonecny12) [inline .structure-config:1]: To resolve infra check fail, please add here also `docs/requirements.txt` file.

**Review by jkonecny12 — CHANGES_REQUESTED** [review]

File: .structure-config (hunk 1 above)
Comment (poncovka) [reply .structure-config]: Fixed.

**Review by rvykydal — APPROVED** [review]
Thank you.

**Review by jkonecny12 — APPROVED** [review]
Looks great to me now. Thanks!

**Pull request merged**